chromiumcontent/chromiumcontent.gyp

```diff
@@ -33,6 +33,7 @@
         '<(DEPTH)/content/content.gyp:content_app_both',
         '<(DEPTH)/content/content_shell_and_tests.gyp:content_shell_pak',
         '<(DEPTH)/net/net.gyp:net_with_v8',
+        '<(DEPTH)/pdf/pdf.gyp:pdf',
```

Comment:
This dependency is only needed on Windows. And `pdf.gyp:pdf` is hard-coded to be `static_library`, we have to deal with it specially in component build.